src/main/java/htsjdk/variant/vcf/VCFMetaDataLines.java

```diff
@@ -0,0 +1,298 @@
+package htsjdk.variant.vcf;
+
+import htsjdk.samtools.util.Log;
+import htsjdk.tribble.TribbleException;
+
+import java.io.Serializable;
+import java.util.*;
+import java.util.stream.Collectors;
+
+//TODO: should this class be public so consumers can use in place of Set<VCFHeaderLine>
```

Comment:
Is any other change planned for the next PRs? If not, maybe it is worthy to make it public in this PR to make people familiar with the API from the beginning...